test/deploy/linux/.net/install/ubuntu20/roles/prepare/tasks/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+- debug:
+    msg: Install ASP.NET Core 5.0 Runtime
+
+- name: Add Microsoft package signing key and add Microsoft package repository for Ubuntu 20.4
+  shell: |
+    wget https://packages.microsoft.com/config/ubuntu/20.04/packages-microsoft-prod.deb -O packages-microsoft-prod.deb
```

Comment:
Is this package available on earlier ubuntu versions? If possible we might like to test on earlier versions.
If it's not supported on earlier ubuntu versions please disregard

Reply:
MSFT has different versions of their .deb repository setup package for different versions of the OS: https://docs.microsoft.com/en-us/dotnet/core/install/linux

If we do want to test multiple Ubuntu versions, I think we should stick to LTS versions, so we would only need to add 16.04 and 18.04.

Reply:
Ho perfect, yeah if we could do with 16.04 and 18.04 that's more than enough.
So maybe we could update the role path to be /ubuntu instead of /ubuntu20, and in the implementation reflect from the /etc/os-release file to pull the correct version number (18.04) for example, and use that for the wget.
BTW, I think we typically use curl instead of wget... But i think curl is not standard on ubuntu? So perhaps fine to keep wget

Reply:
I think there is also an ansible command for curl/wget `uri`, which should do the right thing under the hood. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/uri_module.html

Reply:
Given that our current effort on these test aps is a timeboxed spike, I'd prefer to hold off on trying to test different versions.